### PR TITLE
refactor: Improve style setup

### PR DESCRIPTION
**Pull request by Dignifiedquire**

### CODE REVIEW

File: app/styles/bootstrap.less [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+@import "../../node_modules/bootstrap/less/button-groups";
+@import "../../node_modules/bootstrap/less/input-groups";
+@import "../../node_modules/bootstrap/less/navs";
+@import "../../node_modules/bootstrap/less/navbar";
+@import "../../node_modules/bootstrap/less/breadcrumbs";
+@import "../../node_modules/bootstrap/less/pagination";
+@import "../../node_modules/bootstrap/less/pager";
+@import "../../node_modules/bootstrap/less/labels";
+@import "../../node_modules/bootstrap/less/badges";
+@import "../../node_modules/bootstrap/less/jumbotron";
```
Comment (diasdavid) [inline app/styles/bootstrap.less]: we need `jumbotron` and all the others here?

File: app/scripts/app.js [hunk 2]
```diff
@@ -3,6 +3,8 @@ import ReactDOM from 'react-dom'
 import Router from 'react-router'
 import routes from './routes'
 
+require('../styles/app.less')
+
 if (process.env.NODE_ENV !== 'production') {
   window.uiDebug = require('debug')
```
Comment (diasdavid) [inline app/scripts/app.js]: this could be just `window.debug` so that it becomes clear that it is just our debugging thing.

**Comment by diasdavid** [comment]
overall LGTM :)

File: app/styles/bootstrap.less (hunk 1 above)
Comment (Dignifiedquire) [inline app/styles/bootstrap.less]: This is a 1:1 copy of the requires from bootstrap itself, I have not removed any unused code yet.

File: app/scripts/app.js (hunk 2 above)
Comment (Dignifiedquire) [inline app/scripts/app.js]: That depends if we want to bundle the whole `debug` module, as the current build system will actually not bundle with this setup

File: app/styles/bootstrap.less (hunk 1 above)
Comment (Dignifiedquire) [inline app/styles/bootstrap.less]: Moved to https://github.com/ipfs/webui/issues/153

File: app/scripts/app.js (hunk 2 above)
Comment (Dignifiedquire) [inline app/scripts/app.js]: Also, that will be improved when we do #151

**Pull request merged**